### ISSUE TITLE
Did it use noisy image?

### ISSUE THREAD

**Issue by SparkElf** [JinYize/self2self_pytorch#2]
Hello,I would be appereciate if you could explain why the original self2self.py didn't use noisy image to train the model and if I used the noisy image , the result went wrong：the image seemd too white than the ground truth.
![image](https://user-images.githubusercontent.com/58362707/213103021-25d79e8f-9ae0-4929-a79b-a2c1db9bedae.png)
![image](https://user-images.githubusercontent.com/58362707/213103148-5ce8ef98-83b3-49c7-b770-c8445e17e02a.png)

**Comment by hzy-del** [comment]
> Hello,I would be appereciate if you could explain why the original self2self.py didn't use noisy image to train the model and if I used the noisy image , the result went wrong：the image seemd too white than the ground truth. ![image](https://user-images.githubusercontent.com/58362707/213103021-25d79e8f-9ae0-4929-a79b-a2c1db9bedae.png) ![image](https://user-images.githubusercontent.com/58362707/213103148-5ce8ef98-83b3-49c7-b770-c8445e17e02a.png)

I also have this problem.

**Comment by yuji-byun** [comment]
> > Hello,I would be appereciate if you could explain why the original self2self.py didn't use noisy image to train the model and if I used the noisy image , the result went wrong：the image seemd too white than the ground truth. ![image](https://user-images.githubusercontent.com/58362707/213103021-25d79e8f-9ae0-4929-a79b-a2c1db9bedae.png) ![image](https://user-images.githubusercontent.com/58362707/213103148-5ce8ef98-83b3-49c7-b770-c8445e17e02a.png)
> 
> I also have this problem.

I also have thin problem

**Comment by QilnnGuo** [comment]
> Hello,I would be appereciate if you could explain why the original self2self.py didn't use noisy image to train the model and if I used the noisy image , the result went wrong：the image seemd too white than the ground truth. ![image](https://user-images.githubusercontent.com/58362707/213103021-25d79e8f-9ae0-4929-a79b-a2c1db9bedae.png) ![image](https://user-images.githubusercontent.com/58362707/213103148-5ce8ef98-83b3-49c7-b770-c8445e17e02a.png)

Recently I also tried to reproduce the code, and I think I could answer your problem. You may notice that the author here used the mask for the input with multiplier 0.7 when training, which is unnecessary because the original tensorflow version used tf.nn.dropout(x, keep_drop) which will apply a multiplier 1/keep_drop. And the reason for your brighter output may be caused by that: the code here did not apply the multiplier 0.7 when evaluating despite training input.

**Comment by GCL911** [comment]
> > Hello,I would be appereciate if you could explain why the original self2self.py didn't use noisy image to train the model and if I used the noisy image , the result went wrong：the image seemd too white than the ground truth. ![image](https://user-images.githubusercontent.com/58362707/213103021-25d79e8f-9ae0-4929-a79b-a2c1db9bedae.png) ![image](https://user-images.githubusercontent.com/58362707/213103148-5ce8ef98-83b3-49c7-b770-c8445e17e02a.png)
> 
> Recently I also tried to reproduce the code, and I think I could answer your problem. You may notice that the author here used the mask for the input with multiplier 0.7 when training, which is unnecessary because the original tensorflow version used tf.nn.dropout(x, keep_drop) which will apply a multiplier 1/keep_drop. And the reason for your brighter output may be caused by that: the code here did not apply the multiplier 0.7 when evaluating despite training input.

thank you. very useful!

**Comment by Darkpppp** [comment]
hello，did the problem“the image seemd too white than the ground truth.” happend when training 256*256 RGB images?
